drink/src/runtime.rs

```diff
@@ -2,53 +2,12 @@
 //! `drink` with any runtime that implements the `Runtime` trait.
 
 pub mod minimal;
-pub mod pallet_contracts_debugging;
-
 pub use frame_metadata::RuntimeMetadataPrefixed;
-use frame_support::sp_runtime::{traits::Dispatchable, Storage};
-use frame_system::pallet_prelude::BlockNumberFor;
 pub use minimal::MinimalRuntime;
+pub mod pallet_contracts_debugging;
 
 /// The type of an account identifier.
 pub type AccountIdFor<R> = <R as frame_system::Config>::AccountId;
 
 /// The type of a hash.
 pub type HashFor<R> = <R as frame_system::Config>::Hash;
-
-/// A runtime to use.
```

Comment:
This PR remove the Runtime trait and move the functionality to the `Sandbox` and the `SandboxConfig` trait. This let us bring a Runtime built in another crate and use it as the runtime of our Sandbox.